### ISSUE TITLE
Mac Binary

### ISSUE THREAD

**Issue by jlarmstrongiv** [tajmone/name-that-color#1]
If you have a mac binary that you would like to test, I would gladly try to run it on my machine :) unfortunately, I do not have purebasic

**Comment by tajmone** [comment]
I wish I had ... I'll need someone with macOs and PureBASIC. Also, it would have to be the specific version of PureBASIC used for the project (PureBASIC v5.42) because NTC currently creates an Ascii binary, and the newer versions of PureBASIC no longer support Ascii binaries (only Unicode).

I'm supposed to update the source code to make it work with Unicode binary, but there are some problems with Unicode strings in STDOUT and STDERR on Windows (Ascii chars are rendered with a following white space character when using redirection and piping). 

I haven't yet worked out a safe way to output Ascii strings to STDOUT and STDERR with Unicode binaries. But hopefully I'll find a decent solutions and update the code.

You could try to download the demo version of PureBASIC for free. It has some limitations (no DLL, max 800 lines of code, or something like that). Maybe you can compile it with it:

http://www.purebasic.com/download.php

... but chances are that the demo version is going to be the laster release of PureBASIC, so it will only allow Unicode binaries.

Another possible solution would be to ask on the [PureBASIC forums](http://www.purebasic.fr/english/) if some Mac user is willing to compile it for us. It would be very nice to have binary bersions for all 3 OSs supported.

**Comment by jlarmstrongiv** [comment]
I had tried downloading purebasic and getting it to work, but either because of what you described or user-error, I wasn’t able compile it.  ¯\\\_(ツ)\_/¯

**Comment by tajmone** [comment]
Yes, must be because of the version. If I get around to find some PureBASIC user with Mac who can compile it I'll let you know via PM!
